docs/design/usage-based-scheduling.md

```diff
@@ -92,31 +97,124 @@ Finally, there should a model to balance multiple factors with weight and calcul
 | ...                       |   ...           |    ...           |
 |                           |                 |                  |
 
-
-### Prometheus rule configuration
-The node-exporter is used to monitor the node real-time usage, from which the Prometheus collect the data and aggregate according to the rules. Following Prometheus rules are needed to configured as a example in order to get cpu_usage_avg_5m,cpu_usage_max_avg_1h,cpu_usage_max_avg_1d,mem_usage_avg_5m,mem_usage_max _avg_1h,mem_usage_max_avg_1d etc. 
+### Configuration and usage of different monitoring systems
+The monitoring data of Volcano usage can be obtained from "Prometheus", "Custom Metrics Api" and "Eleasticsearch", where the corresponding type of "Custom Metrics Api" is "prometheus_adapt".
+#### Prometheus
+Scheduler Configuration:
 ```
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-    name: example-record
-spec:
-    groups:
-      - name: cpu_mem_usage_active
-        interval: 30s
-        rules:
-        - record: cpu_usage_active
-          expr: 100 - (avg by (instance) (irate(node_cpu_seconds_total{mode="idle"}[30s])) * 100)
-        - record: mem_usage_active
-          expr: 100*(1-node_memory_MemAvailable_bytes/node_memory_MemTotal_bytes)
-      - name: cpu-usage-1m
-        interval: 1m
-        rules:
-        - record: cpu_usage_avg_5m
-          expr: avg_over_time(cpu_usage_active[5m])
-      - name: mem-usage-1m
-        interval: 1m
-        rules:
-        - record: mem_usage_avg_5m
-          expr: avg_over_time(mem_usage_active[5m])
+actions: "enqueue, allocate, backfill"  
+tiers:
+  - plugins:
+      - name: priority
+      - name: gang
+      - name: conformance
+      - name: usage  # usage based scheduling plugin
+        enablePredicate: false  # If the value is false, new pod scheduling is not disabled when the node load reaches the threshold. If the value is true or left blank, new pod scheduling is disabled.
+        arguments:
+          usage.weight: 5
+          cpu.weight: 1
+          memory.weight: 1
+          thresholds:
+            cpu: 80    # The actual CPU load of a node reaches 80%, and the node cannot schedule new pods.
+            mem: 70    # The actual Memory load of a node reaches 70%, and the node cannot schedule new pods.
+  - plugins:
+      - name: overcommit
+      - name: drf
+      - name: predicates
+      - name: proportion
+      - name: nodeorder
+      - name: binpack
+metrics:                               # metrics server related configuration
+  type: prometheus                     # Optional, The metrics source type, prometheus by default, support "prometheus", "prometheus_adapt" and "elasticsearch"
+  address: http://192.168.0.10:9090    # Mandatory, The metrics source address
+  interval: 30s                        # Optional, The scheduler pull metrics from Prometheus with this interval, 30s by default
+  ```
+#### Custom Metrics Api

```

Comment:
please add notes that prometheus adaptor is recommended.

Reply:
done